### DISCUSSION TITLE
Feed rate increasing when gcode is running

### DISCUSSION THREAD

**Discussion by TechEkansh** [winder/Universal-G-Code-Sender#2229] [Help]
Hello,
I am using 3 28byj-48 stepper motors with uln2003 motor driver to make a small CNC, I have uploaded GRBL to the arduino uno and using UGS to send gcode to the arduino.
when controlling the motor with jog controller ,it is working (at feed rate 100) but when running the gcode the feed rate gets increased to 500 (note: feed rate only increase when the z axis is up, otherwise it is fine).
please can someone tell me how to fix this issue, as I have very little knowledge about UGS and don't know how to fix it.

**Comment by breiler** [comment]
The jog controller has its own feed rate settings in the jog controller window.

If the feed rate is increasing it is likely because your gcode that you are sending tells it to. Can you attach the gcode file?
